nextcord/application_command.py

```diff
@@ -694,12 +694,40 @@ def from_callback(
             if option_class:
                 first_arg = True
                 typehints = typing.get_type_hints(self.callback)
+                callback_params = signature(self.callback).parameters
+                possible_options: Set[str] = set()
+
+                for i, param in enumerate(callback_params.values()):
+                    if (
+                        param.annotation is param.empty or param.annotation is Interaction
```

Comment:
This should use `issubclass` as well as `is` for people who roll with a custom `Interaction` object. (It happens, we should support it)

```py
                    if (
                        param.annotation is param.empty or param.annotation is Interaction or issubclass(param.annotation, Interaction)
                    ) and i in (0, 1):
                        # self or interaction parameter, ignore
                        continue

                    possible_options.add(param.name)
```

Resolves the tests for me locally

Reply:
Advice for next time: you could put a suggestion in that review comment so I could apply the changes immediately